framework/index.js

```diff
@@ -20,4 +20,3 @@ export * from './components/ecl-navigation/ecl-navigation-menus/megamenu';
 export * from './components/ecl-tables/ecl-tables';
 export * from './components/ecl-tabs/tabs';
 // export * from './components/ecl-timelines/timelines';
-// export * from './components/ecl-profile-topbars/profile-topbars';
```

Comment:
Actually it contained a console statement to refactor the js. Since the expandable component can be simply instantiated via the local package js file, I did that directly there without this file.